.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
.... so you can do this with `sudo`, but it'll run it on older infrastructure that will take a while. When you set `sudo: false`, it'll run it in faster Linux containers.